### ISSUE TITLE
Exclude server side injected parameters for spring mvc controller methods.

### ISSUE THREAD

**Issue by jvmlet** [EvidentSolutions/apina#54]
Please exclude known server-side injected parameter types from being generated as ts types.
Consider the controller's method with `HttpServletRequest` as parameter :

````java
@RestController
public class PortalController {
        @GetMapping(path  = "/foo")
    public Foo environmentSettings(HttpServletRequest request) {
 ....
    }
}
````

The generated `ts` file contains generated `HttpServletRequest` type, together with all it's fields : `Cookie`,`Header`,`HttpSession` etc.

**Comment by komu** [comment]
Thanks, that's certainly a regression and Apina shouldn't do that. I'll take a look into that.

**Comment by komu** [comment]
I just released `0.12.3` which should fix this. Thanks again for the report!

**Comment by jvmlet** [comment]
> Will it also strip down the  parameters injected by custom resolvers :-) ?

I see,  you are modeling only parameters with `REQUEST_BODY`, `REQUEST_PARAM` and `PATH_VARIABLE` annotations.
 Thanks, great tool.
